setup.py

```diff
@@ -51,7 +51,7 @@ def find_version(*file_paths):
         'asn1crypto>=1.4.0',
         'cbor2>=5.4.2.post1',
         'cryptography>=41.0.1',
-        'pydantic>=1.10.11,<2.0a0',
```

Comment:
Does `pydantic-core` needs to be defined here? I see it got added in **requirements.txt** but not here which is why I ask.

Reply:
It does not. It's a dependency of pydantic, not something that's used directly here. I've updated requirements.txt using `pip freeze > requirements.txt` which is how I'm used to doing it. Happy to remove/add anything you want.